### ISSUE TITLE
[Question] Empty build commit

### ISSUE THREAD

**Issue by jamesst20** [pineappleEA/pineapple-src#107]
Hi,

Just wondering what it means when there are empty build commit like 3555, 3556.

Does it just mean nothing changed but a new build was triggered with not changes?

**Comment by MGThePro** [comment]
Yes. New builds get triggered automatically, even when there is no real change, like a PR tagged early-access-merge gets merged into master. Technically the repo changed, but the PR was present before and after the merge, so no real change happened.
 There might be other instances where this could happen, but it doesn't really matter.
This is on yuzu's end anyways, so you get the same "no change builds" as a patron, you just wouldn't notice it as easily.
